rebuild_rust_sdk.sh

```diff
@@ -42,6 +42,9 @@ fi
 # replace uniffi version to one that works with uniffi-bindgen-go
 echo 'building matrix-sdk-ffi...';
 cd $RUST_SDK_DIR;
+cp Cargo.toml Cargo.toml.backup
+cp Cargo.lock Cargo.lock.backup
+trap "mv $RUST_SDK_DIR/Cargo.toml.backup $RUST_SDK_DIR/Cargo.toml; mv $RUST_SDK_DIR/Cargo.lock.backup $RUST_SDK_DIR/Cargo.lock" EXIT INT TERM
 sed -i.bak 's/uniffi =.*/uniffi = "0\.25\.3"/' Cargo.toml
 sed -i.bak 's^uniffi_bindgen =.*^uniffi_bindgen = { git = "https:\/\/github.com\/mozilla\/uniffi-rs", rev = "0a03b713306d6ce3de033157fc2ce92a238c2e24" }^' Cargo.toml
 sed -i.bak 's#matrix-sdk-crypto = {#matrix-sdk-crypto = {features = ["_disable-minimum-rotation-period-ms"],#' Cargo.toml
```

Comment:
given `-i.bak` gives you a backup, maybe you could save the separate `cp` operation. You would have to combine the sed commands into one, but that should be fine?

Reply:
I'm using `-i.bak` in particular for portability reasons. FreeBSD/Mac demands `-i` has a suffix, but GNU does not. It's not standardised at all, so I'm reluctant to rely on it for providing backups.

Reply:
Surely the fact that you're successfully using it means that it must be accepted? Are you really concerned that there might be implementations of sed that accept a suffix to `-i` but ignore it?

Not that this really matters, it just feels odd to duplicate the backup